docs/indepth/installation.rst

```diff
@@ -215,7 +215,7 @@ Using Amazon Web Services
 
```

Comment:
OK, I'm currently editing the same page to add Ubuntu instructions, so I might drop that tid-bit in.